bincrafters_conventions/actions/update_travis_url.py

```diff
@@ -0,0 +1,7 @@
+def update_travis_url(main, file):
```

Comment:
Could we please rename this function + file to `update_readme_travis_url`? I would like to have them prefix with the category of file(s) it is manipulating

Reply:
yes, it sounds better. I'll update it.

Reply:
done!